apps/common/components/ListHero.tsx

```diff
@@ -170,4 +170,4 @@ function ListHero<T extends string>({
 		</div>
 	);
 }
-export default ListHero;
+export {ListHero};
```

Comment:
I know it's a chore, but can we just export the function inline, rather than at the end of the file?